### PR TITLE
VM name change fix

### PR DESCRIPTION
**Pull request by sshveta**
Vm name Change - underscore is removed.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 66ce3ef8e2b53f790f36dd436cfab866b975ca92:

`cfme/services/catalogs/catalog_item.py`:
- :red_circle: Line 18:1: **F401** *'BZ' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a45744956249fe3deee8a6ec40c4678b917e7d2f:

`cfme/services/catalogs/catalog_item.py`:
- :red_circle: Line 18:1: **F401** *'BZ' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 036c9ee25b8714477ff844cdad713e5da1e61582:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by sshveta** [comment]
5.6 Local test result:

py.test --long-running -v cfme/tests/services/test_myservice.py 
Loading templates from trackerbot                                                                                                                                                                                  
no test annotation found in None
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.5, pytest-2.9.2, py-1.4.31, pluggy-0.3.1 -- /home/sshveta/Work/cfme_vir_env/bin/python2
cachedir: .cache
rootdir: /home/sshveta/Work/cfme_vir_env/cfme_tests, inifile: 
plugins: wait-for-1.0.8
collected 6 items 

Appliance's streams: [5.6, downstream]

cfme/tests/services/test_myservice.py::test_retire_service[vsphere55] PASSED
cfme/tests/services/test_myservice.py::test_retire_service_on_date[vsphere55] PASSED
cfme/tests/services/test_myservice.py::test_crud_set_ownership_and_edit_tags[vsphere55] PASSED
cfme/tests/services/test_myservice.py::test_download_file[vsphere55-Text] PASSED
cfme/tests/services/test_myservice.py::test_download_file[vsphere55-CSV] PASSED
cfme/tests/services/test_myservice.py::test_download_file[vsphere55-PDF] PASSED

**Comment by dajoRH** [comment]
Lint report for commit ba2eb80ef5369d4b8f1288ef47b08ce9e33b9d54:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/services/catalogs/catalog_item.py [hunk 1]
```diff
@@ -171,9 +170,6 @@ def create(self):
             tabstrip.select_tab("Catalog")
             template = template_select_form.template_table.find_row_by_cells({
                 'Name': self.catalog_name,
-                # HACK to workaround BZ1390209
```
Comment (mshriver) [inline cfme/services/catalogs/catalog_item.py]: :+1:

File: cfme/tests/services/test_myservice.py [hunk 2]
```diff
@@ -45,7 +46,9 @@ def myservice(setup_provider, provider, catalog_item, request):
     Metadata:
         test_flag: provision
     """
-    vm_name = catalog_item.provisioning_data["vm_name"] + "_0001"
+    vm_name = version.pick({
+        version.LOWEST: catalog_item.provisioning_data["vm_name"] + '_0001',
+        '5.7': catalog_item.provisioning_data["vm_name"] + '0001'})
```
Comment (mshriver) [inline cfme/tests/services/test_myservice.py]: Can we just use the new syntax for earlier versions, or does using '_0001' in 5.6 do something worthwhile for us?

**Pull request merged**